course_contents/2_intro_to_python/notes/01. Conditions.md

```diff
@@ -0,0 +1,101 @@
+### The `if` statement
+- the `if` statement is used to check if a conditional is `True`
+- syntax for writing the `if` statement:
+    1. use the `if` keyword at the beginning of the statement
+    2. follow the keyword with a correctly written conditional
+    3. finish the expression with colon 
+    4. block of code that will be executed if the statement passes - should be indented (usually by 4 spaces)
+
+An example of proper writing of the `if` statement:
+
+```py
+friend = "Rolf"
+name = input("Enter your name: ")
+if name == friend:
+    print(f"{name} is a friend")
+```
+
+Two examples of incorrect writing of the `if` statement:
+
+```py
+if name = friend:
+    print(f"{name} is a friend")
+```
+```py
+if name == friend:
+print(f"{name} is a friend")
+```
+**Extra resources**
+- [What is an if statement?](https://python.tecladocode.com/2_countries_visited/2_if_statements.html#what-is-an-if-statement)
+
+### The compound `if` statement - `elif` keyword
+- `elif` keyword is used for chaining multiple if statements
+- `elif` block will be entered only if the previous `if`/`elif` block doesn't get entered
```

Comment:
I would say `isn't executed` instead of `doesn't get entered`

Reply:
The block technically is executed, the expression must be checked in order to proceed further, but the statement inside of that block is not executed, which I shortened to "doesn't get entered". Once again, I did this to shorten the text as much as possible